tools/onnx2script.py

```diff
@@ -0,0 +1,41 @@
+# -------------------------------------------------------------------------
```

Comment:
I would add usage in the module docstring

Reply:
Done